src/Scriban/Functions/StringFunctions.cs

```diff
@@ -971,10 +971,66 @@ public static string PadLeft(string text, int width)
         /// ```html
         /// hello     world
         /// ```
-        /// </remarks> 
+        /// </remarks>
         public static string PadRight(string text, int width)
         {
             return (text ?? string.Empty).PadRight(width);
         }
+
+        /// <summary>
+        ///  Encodes a string into an array of UTF-8-encoded bytes.
+        /// </summary>
+        /// <param name="text">The string to encode</param>
+        /// <returns>The encoded bytes</returns>
+        public static byte[] Utf8Encode(string text)
```

Comment:
this will require to regenerate the builtin delegates (run `Scriban.CodeGen` in the project)

Reply:
It doesn't. Unless something is wrong... Wait a minute.

![image](https://user-images.githubusercontent.com/12659251/52492475-f203b800-2bd1-11e9-92e0-441144559459.png)

Reply:
Is it normal? Should new code be generated? Did I break anything?

Reply:
I don't know,, maybe the problem is not handling this kind of parameter types, it is the first time that there is a `byte[]` so maybe there was an exception when you ran it?

Reply:
No, there was no exception. The process returned 0.

Reply:
I don't know, you will have to debug, maybe something different that doesn't work well on your setup, or a bug in the program itself